### DISCUSSION TITLE
transcript with no extracted_wisdom

### DISCUSSION THREAD

**Discussion by petrelli82** [danielmiessler/Fabric#751] [Q&A]
I keep getting a four digit code like: 
[1] 2086
then at the end it says:
I'm sorry, but it seems there was no input provided for me to process. Could you please provide the text content or details you'd like me to analyze?
[1]+  Done 
Then it just gives me the transcript of the video. I’m trying to extract wisdom. How do I fix this?

**Comment by stuboo** [comment]
Can you give us the exact command you're using so we can try to reproduce the error?
